pkg/daemon/ceph/client/mirror.go

```diff
@@ -362,6 +362,40 @@ func ListSnapshotSchedulesRecursively(context *clusterd.Context, clusterInfo *Cl
 	return snapshotSchedulesRecursive, nil
 }
 
+// ListRadosNamespacesInPool lists the rados namespaces in a pool
+func ListRadosNamespacesInPool(context *clusterd.Context, clusterInfo *ClusterInfo, poolName string) ([]string, error) {
```

Comment:
can you please move this to `ceph/client/radosnamspace.go` there we where we are keeping all the CLI calls for rados namespace.